src/store.js

```diff
@@ -100,7 +100,7 @@ export class Store {
         handler(payload)
       })
     })
-    this._subscribers.forEach(sub => sub(mutation, this.state))
```

Comment:
I think we also need to do the same thing for `_actionSubscribers` in `dispatch` method.
https://github.com/vuejs/vuex/blob/dev/src/store.js#L132

Reply:
Good catch, I've shared it through a new `notifySubscribers` function.